cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -11,7 +12,7 @@
 
 
 @pytest.fixture(scope='module')
-def enabled_embedded_appliance(appliance):
+def embedded_appliance(appliance):
```

Comment:
Optional: 
but best place of this fixture is `global` as per me. what you think?... lets take advantage of this fixture to other contributes and may be same FA owner as well.

Reply:
Sure, not going to move it in this PR though :)

Certainly, defining basic fixtures to be consumed globally would be much more efficient.